modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+By configuring delegated image scanning, you can scan images by using the secured clusters instead of the Central services.
+
+A new delegated registry configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries. 
```

Comment:
I'm not that familiar with this part of the docs. I'm assuming that capitalized things in running text like "Sensor" and "Scanner-slim" are proper product names.

Reply:
Those are the major components of ACS